src/converter/FlameGraph.java

```diff
@@ -115,36 +117,32 @@ public void dump(PrintStream out) {
         mintotal = (long) (root.total * minwidth / 100);
         int depth = mintotal > 1 ? root.depth(mintotal) : this.depth + 1;
 
-        out.print(applyReplacements(HEADER,
-                "{title}", title,
-                "{height}", Math.min(depth * 16, 32767),
-                "{depth}", depth,
-                "{reverse}", reverse));
+        String tail = getResource("/flame.html");
+
+        tail = printTill(out, tail, "/*height:*/300px/**/");
```

Comment:
Why having `/**/` at the end, if you look for the entire template anyway?
IMO, `/*height:*/300px` would be slightly cleaner.

Reply:
Changed for ```/*height:*/300``` - it removes the need to add "px" suffix.